### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by lynnbendixsen**
Added a ToC and Linux and other items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,71 @@
...
-* sudo apt-get update
-* sudo apt-get upgrade
-* sudo apt-get install -y libindy
-* sudo apt install python3.8 -y
-* sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.6 1
-* sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 2
-* sudo apt install python3-pip git
-* pip3 install --upgrade pip
-* pip3 install aiohttp base58 indy Python3-indy asyncio
+[**Docker Instructions**](#docker)
```
Comment (lynnbendixsen) [inline README.md:1]: will it look nicer to move the title to the top, and then have the TOC?

File: README.md [hunk 2]
```diff
@@ -1,67 +1,71 @@
...
+[**Alternate Setup Instructions**](#alternate)  
+[Linux](#linux)  
+[Windows](#windows)  
+[Mac](#mac)  
+[**Running Instructions**](#running)
 
+# Author-Endorser-Wizards
+Python program to walk through the author and endorser workflow to create schemas and cred defs.
+
+## Docker Instructions <a id="docker"></a>
```
Comment (lynnbendixsen) [inline README.md:13]: Rename to docker setup instructions?  Newbie coming to the site might be confused as to why there is even a docker setup here.  We should probably add some explanatory steps here or above as a quick explanation for help with navigating?
"The easiest way to get up and running the Author and Endorser wizards is to use the docker setup...(etc.)"  Probably the preceeding paragraph should be expanded to 2-3 more sentences of explanation.

File: README.md [hunk 3]
```diff
@@ -1,67 +1,71 @@
...
+
+### Running the Endorser <a id="endorser"></a>
```
Comment (lynnbendixsen) [inline README.md:15]: this should say  'Endorser Wizard'

**Review by lynnbendixsen — COMMENTED** [review]
This all looks really good!  I added a few comments that  could help enhance the first time  user experience, but I am not sure that I am commenting on the final copy that you submitted. It really looks like this was the first commit and not the last one, sorry for the troubles, git is giving me troubles again :) I am unable to do anything but comment because I submitted the PR.

File: README.md (hunk 1 above)
Comment (mikekebert) [reply README.md:1]: Agreed, let's move the title and one line introduction above the TOC.

File: README.md (hunk 2 above)
Comment (mikekebert) [reply README.md:13]: I think it should say "Docker Setup Instructions." We can more depth when we get a chance.

File: README.md (hunk 3 above)
Comment (mikekebert) [reply README.md:15]: Agreed.

File: README.md (hunk 1 above)
Comment (mikekebert) [reply README.md:1]: Agreed, let's move the title and one line introduction above the TOC.

**Review by mikekebert — CHANGES_REQUESTED** [review]
Overall, the structure and content look good. I won't have time to actually try the instructions. Let's resolve Lynn's 3 comments and then we can merge this.

**Review by lynnbendixsen — COMMENTED** [review]
Looks great!

**Review by mikekebert — APPROVED** [review]
Good job

**Pull request merged**